### PR TITLE
Add dh-{stage,prod}-trino in kustomization.yaml

### PR DESCRIPTION
**Pull request by rimolive**
## This Pull Request implements

Add dh-{stage,prod}-trino in kustomization.yaml

## If migrating an Application to ArgoCD
- [X] I have completed the list of action items on [this page](https://aicoe.github.io/aicoe-cd/get_argocd_to_manage_your_app/)

### CODE REVIEW

File: manifests/overlays/prod/applications/data_hub/kustomization.yaml [hunk 1]
```diff
@@ -21,3 +21,5 @@ resources:
 - dh-prod-thanos-extractor.yaml
 - dh-stage-rhods-catalog.yaml
 - dh-stage-rhods-objects.yaml
+- dh-stage-trino
+- dh-prod-trino
```
Comment (anishasthana) [inline manifests/overlays/prod/applications/data_hub/kustomization.yaml:25]: Missing new line :-)

**Review by anishasthana — CHANGES_REQUESTED** [review]
Missing a new line

**Review by gmfrasca — COMMENTED** [review]
/lgtm

**Review by gmfrasca — COMMENTED** [review]
/lgtm again

**Review by anishasthana — APPROVED** [review]
/lgtm

**Pull request merged**